js/gratipay/team.js

```diff
@@ -61,43 +56,50 @@ Gratipay.team = (function() {
             if (member.take === member.max_this_week)
                 increase = 'max';
 
-            if (i < nmembers)
-                rows.push(Gratipay.jsonml(
-                    [ 'tr'
-                    , ['td', {'class': 'n'}, (i === nmembers ? '' : nmembers - i)]
-                    , ['td', ['a', {'href': '/'+member.username+'/'}, member.username]]
-                    , ['td', {'class': 'figure last_week'}, num(member.last_week)]
-                    , ['td', {'class': 'figure take ' + increase}, drawMemberTake(member)]
-                    , ['td', {'class': 'figure balance'}, num(member.balance)]
-                    , ['td', {'class': 'figure percentage'}, perc(member.percentage)]
-                     ]
-                ));
-            else if (nmembers > 0)
-                rows.push(Gratipay.jsonml(
-                    [ 'tr'
-                    , ['td']
-                    , ['td']
-                    , ['td']
-                    , ['td', {'class': 'figure take'}, num(member.take)]
-                    , ['td', {'class': 'figure balance'}, num(member.balance)]
-                    , ['td', {'class': 'figure percentage'}, perc(member.percentage)]
-                     ]
-                ));
+            rows.push(Gratipay.jsonml(
+                [ 'tr'
+                , ['td', {'class': 'n'}, (i === nmembers ? '' : nmembers - i)]
+                , ['td', ['a', {'href': '/~'+member.username+'/'}, member.username]]
+                , ['td', {'class': 'figure last_week'}, num(member.last_week)]
+                , ['td', {'class': 'figure take ' + increase}, drawMemberTake(member)]
+                , ['td', {'class': 'figure balance'}, num(member.balance)]
+                , ['td', {'class': 'figure percentage'}, perc(member.percentage)]
+                 ]
+            ));
+        }
+
+        if (nmembers === 0) {
+            rows.push(Gratipay.jsonml(
+                ['tr', ['td', {'colspan': '6', 'class': 'no-members'}, "No members"]]
+            ));
+        } else {
+            rows.push(Gratipay.jsonml(
+                [ 'tr'
+                , {'class': 'totals'}
+                , ['td']
+                , ['td']
+                , ['td']
+                , ['td', {'class': 'figure take'}, num(available - member.balance)]
+                , ['td', {'class': 'figure balance'}, num(member.balance)]
+                , ['td', {'class': 'figure percentage'}, perc(member.balance / available)]
```

Comment:
Hmm ... without the sidebar there's plenty of room for the table, but I think we're supposed to have the sidebar, even though we don't right now (the team name is missing as well):

![screen shot 2016-08-04 at 2 15 05 pm](https://cloud.githubusercontent.com/assets/134455/17413194/368bece8-5a4e-11e6-85b3-90f8cdf05bba.png)